subprojects/android/src/main/java/org/mockito/android/internal/creation/AndroidByteBuddyMockMaker.java

```diff
@@ -1,13 +1,31 @@
 package org.mockito.android.internal.creation;
 
 import org.mockito.internal.creation.bytebuddy.SubclassByteBuddyMockMaker;
+import org.mockito.internal.util.Platform;
 import org.mockito.invocation.MockHandler;
 import org.mockito.mock.MockCreationSettings;
 import org.mockito.plugins.MockMaker;
 
+import static org.mockito.internal.util.StringJoiner.join;
+
 public class AndroidByteBuddyMockMaker implements MockMaker {
 
-    private final MockMaker delegate = new SubclassByteBuddyMockMaker(new AndroidLoadingStrategy());
+    private final MockMaker delegate;
+
+    public AndroidByteBuddyMockMaker() {
+        if (Platform.isAndroid() || Boolean.getBoolean("org.mockito.mock.android")) {
+            delegate = new SubclassByteBuddyMockMaker(new AndroidLoadingStrategy());
+        } else {
+            System.err.println(join(
```

Comment:
OTH maybe `err` is a bit too much, since this code enable the mockito-android to fallback on the core mockmaker ? info level maybe sufficient ?